roles/confluent.control_center/handlers/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - name: restart control center
   include_tasks: tasks/restart_and_wait.yml
+  when:
+    - skip_restarts | default("false") == false
```

Comment:
I think this should be a documented boolean variable - if added at all. Why would you skip restarts - if needed?

Reply:
Ok, I'll that to the documentation ... 

The rationale behind this is that the restart handlers invoke `systemd`, however, in a "rootless" environment [without a systemd user instance] root access is required to execute the restart -- which is unavailable. The user should have the option to skip the restart handler.

Reply:
also wondering if we should have
```
skip_restarts: false
control_center_skip_restarts: "{{skip_restarts}}"
```
This is a pattern we use elsewhere. Do you think there'd be any case where someone would want to isolate a service?

Reply:
In theory, I guess... 

For the use case that we've been working with, a restart requires root access (e.g. for `systemd`), and the working assumption is that none of the machines have root access available, and thus it's more of a global thing. 

I don't think it's that big of a deal to scope it per component in the same way other variables are ... I'll make that change ...